types/parse/index.d.ts

```diff
@@ -785,6 +785,7 @@ declare global {
             subscribe(): Promise<LiveQuerySubscription>;
             toJSON(): any;
             withJSON(json: any): this;
+            withCount(): this;
```

Comment:
```suggestion
            withCount(includeCount?: boolean): this;
```
Should it ship wiht optional bool parameter?
http://parseplatform.org/Parse-SDK-JS/api/3.1.0/Parse.Query.html#withCount

Reply:
right!